### PR TITLE
Remove Gen1 pricing from the template

### PR DESCRIPTION
**Pull request by dubiety**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Remove Azure Maps Gen1 pricing from the pricing tier dropdown
* Update description of pricing tiers

### CODE REVIEW

**Comment by azure-quickstarts** [comment]
@philmea - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@eriklindeman - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

File: quickstarts/microsoft.maps/maps-use-sas/metadata.json [hunk 1]
```diff
@@ -5,6 +5,6 @@
   "description": "This template deploys and Azure Maps account and lists a Sas token based on the provided User Assigned identity to be stored in an Azure Key Vault secret.",
   "summary": "This template deploys and Azure Maps account and lists a Sas token and creates an Azure Key Vault secret with the sas token as the value.",
   "githubUsername": "stack111",
-  "docOwner": "philmea",
-  "dateUpdated": "2021-10-24"
+  "docOwner": "eriklindeman",
```
Comment (stevemunk) [inline quickstarts/microsoft.maps/maps-use-sas/metadata.json:8]: Shouldn't the `docOwner` be _stack111_?

Comment (dubiety) [reply quickstarts/microsoft.maps/maps-use-sas/metadata.json:8]: docOwner can review the PR, so I assign it to Erik. I'm ok to assign to either `stack111` or `eriklindeman`.

Comment (dubiety) [reply quickstarts/microsoft.maps/maps-use-sas/metadata.json]: Updated

**Comment by azure-quickstarts** [comment]
@stack111 - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Review by eriklindeman — APPROVED** [review]

**Pull request merged**